tutorials/subdomain_and_interface_projections.ipynb

```diff
@@ -0,0 +1,525 @@
+{
```

Comment:
<p>I like the explanation, and the figure is really nice. However, the overall text makes it appear that this weighting is only relevant for intensive quantities, and the reader will need to do some interpretation to understand what is different and what is similar for intensive quantities. Ideally, I would have liked to see this explanation given before the pressure example above, with an additional illustration of the projection of intensive quantities.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pmgbergen/porepy/pull/1286/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>